### PR TITLE
Make tests pass on Windows

### PR DESCRIPTION
**Pull request by rmg**
* Use smarter assertions (`t.match(a, b)` vs `t.assert(b.test(a))`) for improved test debugging.
* Avoid `process.exit()` where possible to ensure stdio is flushed, allowing expected error messages to be seen
* Skip SSH based tests on Windows since it is unlikely to have sshd running locally

### CODE REVIEW

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

File: bin/sl-meshctl.js [hunk 1]
```diff
@@ -92,11 +90,11 @@ if (process.env.SSH_PORT) {
 dieIf.url = apiUrl;
 maybeTunnel(apiUrl, sshOpts, function(err, apiUrl) {
   dieIf.url = apiUrl || dieIf.url;
-  dieIf(err);
+  if (err) return dieIf(err);
```
Comment (sam-github) [inline bin/sl-meshctl.js]: https://github.com/strongloop/strong-mesh-models/blob/cleanup-tests/bin/sl-meshctl.js#L827

This reads funny "if error die if error"... The "if" is what `dieIf()` is already supposed to do. I see your problem, you want to return early. `dieIf()` should renamed `die()` if the `if` always has to be done outside the function.

**Comment by sam-github** [comment]
LGTM

File: bin/sl-meshctl.js (hunk 1 above)
Comment (rmg) [inline bin/sl-meshctl.js]: Agreed. I was trying to keep the changes minimal, but I probably overshot.

**Comment by rmg** [comment]
@slnode test please

**Pull request merged**